kytos/core/link.py

```diff
@@ -168,21 +169,22 @@ def get_next_available_tag(
     def make_tags_available(
         self,
         controller,
-        tags: Union[int, list[int]],
+        tags: Union[int, list[int], list[list[int]]],
         link_id,
-        tag_type: str = 'vlan'
-    ) -> tuple[bool, bool]:
+        tag_type: str = 'vlan',
+        check_order: bool = True,
+    ) -> tuple[list[list[int]], list[list[int]]]:
         """Add a specific tag in available_tags."""
         with self._get_available_vlans_lock[link_id]:
             with self.endpoint_a._tag_lock:
                 with self.endpoint_b._tag_lock:
-                    result_a = self.endpoint_a.make_tags_available(
-                        controller, tags, tag_type, False
+                    conflict_a = self.endpoint_a.make_tags_available(
+                        controller, tags, tag_type, False, check_order
```

Comment:
If you could pass the `False` as `use_lock=False`, and `check_order=check_order` that'd be great for immediate code readability

Reply:
Added argument values to call.